.devops/deploy-pipelines.yml

```
@@ -144,9 +144,9 @@ stages:
             parameters:
               azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
               appName: '$(PRODUCTION_APP_NAME)'
-              endpoint: 'https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net/template/info'
+              endpoint: 'https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net/api/v1/cgn/info'
               endpointType: 'private'
-              containerInstanceResourceGroup: 'io-p-rg-common'
+              containerInstanceResourceGroup: 'io-p-rg-cgn'
```

This resource group refers to the container that performs the healthcheck, not the current app
```suggestion
              containerInstanceResourceGroup: 'io-p-rg-common'
```